teams/compiler.toml

```diff
@@ -110,3 +110,8 @@ name = "T-compiler"
 
 [[zulip-groups]]
 name = "T-compiler/meeting"
+include-team-members = false
+extra-people = [
+    "davidtwco",
+    "wesleywiser"
```

Comment:
@rust-lang/compiler who wants to be pinged by `T-compiler/meeting`?

Reply:
+1

Reply:
+1

Reply:
+1

Reply:
+1

Reply:
+1

Reply:
+1

Reply:
Updated with everyone above

@apiraino: we should add to the team meeting agenda a small note that anyone who wants pinged for the meeting can make a PR to add to themselves to this list